src/quantem/tomography/tomography_dataset.py

```diff
@@ -0,0 +1,212 @@
+import torch
+from numpy.typing import NDArray
+from torch._tensor import Tensor
+
+from quantem.core.datastructures.dataset3d import Dataset3d
+from quantem.core.io.serialize import AutoSerialize
+from quantem.core.utils.validators import (
+    validate_array,
+    validate_tensor,
+)
+
+
+class TomographyDataset(AutoSerialize):
```

Comment:
What is the point of having TomographyDataset and TiltSeries?  It seems like these 2 classes do the same thing.